kern/common.h

```diff
@@ -32,6 +32,13 @@ const volatile u64 target_pid = 0;
 // u64 target_pid = 0;
 #endif
 
+//optional Target ErrNo
+#define ERRNO_DEFAULT 128
+#ifndef KERNEL_LESS_5_3
```

Comment:
KERNEL_LESS_5_3   or   KERNEL_LESS_5_2 , typos ?

Reply:
I don't konw KERNEL_LESS_5_3 means, but I think that pid is global flag and errnumber is local flag

Reply:
https://github.com/ehids/ecapture/issues/39

Reply:
and : [.rodata: map create: read- and write-only maps not supported (requires >= v5.2)](https://github.com/cilium/ebpf/issues/628)

Reply:
got it, i will put them together